<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AngpOutputBox</class>
 <widget class="QWidget" name="AngpOutputBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Характеристики горизонтального наведения</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="time_label">
       <property name="text">
        <string>Time:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="time_line"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="dist_label">
       <property name="text">
        <string>Dist:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="dist_line"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="kurs_label">
       <property name="text">
        <string>Kurs:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="kurs_line"/>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="radius_label">
       <property name="text">
        <string>Radius:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="radius_line"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
